setup.cfg

```diff
@@ -0,0 +1,23 @@
+[pycodestyle]
+exclude = .eggs,ENV,build,docs/conf.py,venv
+
+[yala]
+radon mi args = --min C
+pylint args = --disable=too-many-locals,too-few-public-methods,too-many-instance-attributes,too-many-arguments,inconsistent-return-statements,unnecessary-pass,too-many-public-methods,redefined-outer-name,unnecessary-lambda,missing-timeout,import-error,no-name-in-module,attribute-defined-outside-init --ignored-modules=napps.kytos.of_lldp
```

Comment:
`of_lldp` stuff shouldn't be here, probably accidental from when you initially based on `of_lldp` config, please review this, clean it up, and also consider removing certain ignored args here if they aren't needed too.